server/models/attributes/listing.ts

```diff
@@ -8,5 +8,12 @@ export default {
   updatable,
   validAtCreation: updatable.concat([
     'creator',
+    'type',
   ]),
+  type: [ 'work', 'author', 'publisher' ],
```

Comment:
What about keeping `entityTypesByListingType` as the single source of truth for listing types?
```suggestion
  type: Object.keys(entityTypesByListingType),
```

Reply:
done in 885ec0428